modelio.go

```diff
@@ -407,9 +407,9 @@ func (model *Model) DumpToString() (string, error) {
 	}
 
 	output = append(output, "SV\n")
-	if len(model.svCoef)==model.l {
+	if len(model.svCoef)==model.nrClass-1 {
 		for i := 0; i < model.l; i++ {
-			if len(model.svCoef[i]) == model.nrClass-1{
+			if len(model.svCoef[i]) == model.l{
```

Comment:
Here you use `i` as the index of the first dimension. In line 414 you use `j` instead. I think instead of changing the order of the two if statements you just need to swap the indexes in line 414, right?

Reply:
(Also the code is not formatted correctly)